runtime-decompiler/src/main/java/org/jrd/backend/data/Config.java

```diff
@@ -4,26 +4,33 @@
 import org.jrd.backend.core.OutputController;
 
 import java.io.File;
+import java.io.FileReader;
 import java.io.IOException;
 import java.nio.charset.Charset;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.util.ArrayList;
 import java.util.HashMap;
-import java.util.List;
+import java.util.Collections;
+import com.google.gson.Gson;
+
 
 /**
  * Singleton class for storing and retrieving configuration strings.
  */
 public class Config {
 
     private static Config config;
+    private static Gson gson;
```

Comment:
just brainstroming - any reason to have it  class wide ?
Likely no, but is slightly better then local wide:) Can you makeit final?

Reply:
Yeah you're probably right